draft-ietf-core-oscore-groupcomm.md

```diff
@@ -1360,9 +1360,9 @@ For endpoints that support the group mode, the following applies.
 
 * For many constrained IoT devices, it is problematic to support more than one signature algorithm. The following applies with respect to the Signature Algorithm (see {{ssec-common-context-cs-alg}}).
 
-   Less constrained endpoints SHOULD implement both: the EdDSA signature algorithm together with the elliptic curve Ed25519 {{RFC8032}}; and the ECDSA signature algorithm together with the elliptic curve P-256.
+   Less constrained endpoints MUST implement at least one of the following and SHOULD implement both: the EdDSA signature algorithm together with the elliptic curve Ed25519 {{RFC8032}}; the ECDSA signature algorithm together with the elliptic curve P-256.
 
-   Constrained endpoints SHOULD implement: the EdDSA signature algorithm together with the elliptic curve Ed25519 {{RFC8032}}; or the ECDSA signature algorithm together with the elliptic curve P-256.
+   Constrained endpoints MUST implement at least one of the following: the EdDSA signature algorithm together with the elliptic curve Ed25519 {{RFC8032}}; the ECDSA signature algorithm together with the elliptic curve P-256.
```

Comment:
Is there a definition of "less-constrained" to determine which requirement applies to a given endpoint? If not, I'd thought this would be a MUST-one/SHOULD-both.

Reply:
> Is there a definition of "less-constrained" to determine which requirement applies to a given endpoint? 

To answer this specific question:
We generally use RFC 7228 for a definition of "constrained node" (Section 2.1).
(This document has a -[bis][] in IOTOPS, btw.)

[bis]: https://www.ietf.org/archive/id/draft-ietf-iotops-7228bis-01.html